styles/toast.css

```diff
@@ -0,0 +1,17 @@
+.toast-container{
+    display: none;
+    width: 300px;
+    padding: 0.2rem 1rem;
+    margin: 0.5rem;
+    border-radius: 1rem;
+    border: 1px solid var(--primaryBg);
+    background-color: var(--theme50);
+    box-shadow: 5px 5px 10px var(--theme200),-5px -5px 10px var(--theme200);
+    position: fixed;
+    bottom: 3%;
+    right: 1%;
+}
+
+.toast-msg{
+    padding: 0.5rem 0;
+}
```

Comment:
General Comments:
1. All iframe are of different widths so doesn't look uniform
2. You can change color of navbar, it doesn't look good overall
3. Not responsive on 320px,and 360px, I think because of non-uniform width of cards or images
4. Badges are not spaced properly on smaller screens

Reply:
Fixed point 3 & 4
Will look into the first 2 points after some time.